src/test/java/fr/fifiz/training/app/java/BasicWeatherAppApplicationTests.java

```diff
@@ -0,0 +1,35 @@
+package fr.fifiz.training.app.java;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import org.junit.jupiter.api.Test;
+
+/**
+ * Unit test for BasicWeatherAppApplication simple App.
+ *
+ * @author bertrand
+ */
+public class BasicWeatherAppApplicationTests {
+
+	/**
+	 * Rigorous Test :-)
+	 */
+	@Test
+	public void contextLoads() {
+		assertTrue(true);
+	}
+
+	@Test
+	public void testMain() throws IOException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/bdelion/basic-weather-app/pullRequest?prid=4408921)